index.html

```diff
@@ -242,7 +242,7 @@ <h6>Made with <i class="fa fa-heart fa-fw"></i> in Luxembourg & Switzerland
     <script src="js/jquery.easing.min.js"></script>
     <script src="js/jquery.countdown.min.js"></script>
     <script src="js/device.min.js"></script>
-    <script src="js/form.min.js"></script>
+    <!-- <script src="js/form.min.js"></script> -->
```

Comment:
@marcrowindo I had to disable the form logic from the template, otherwise it captures the submit event and tries to reach the mail php thing. So don't forget to disable this script in all html pages where the form should work! Not only index like I did here